docs/user-guide/SettingsFile.md

```diff
@@ -37,6 +37,49 @@ Path to your key file in a txt file.
 
 If provided and valid, RESTler will attempt to use it during the SSL handshake.
 
+### Authentication settings dict (default empty)
```

Comment:
>Authentication settings  [](http://example.com/codeflow?start=4&length=24)

authentication: dict (default empty)

These are in the format <name of setting>: type (default)